### PR TITLE
Updated CoreOS bare metal docs for 0.15.0

### PR DESCRIPTION
**Pull request by hvolkmer**
Also added more consistent placeholder names, info how to replace them and added more debugging commands

### CODE REVIEW

File: docs/getting-started-guides/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Mesos/GCE      |              |        |             | [docs](../../docs/getting
 AWS            | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community                    | Uses K8s version 0.11.0
 GCE            | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community (@kelseyhightower) | Uses K8s version 0.11.0
 Vagrant        | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community (@pires)           | Uses K8s version 0.11.0
-Bare-metal (Offline) | CoreOS       | CoreOS | flannel      | [docs](../../docs/getting-started-guides/coreos/bare_metal_offline.md) | Community([@jeffbean](https://github.com/jeffbean))    | K8s v0.10.1
+Bare-metal (Offline) | CoreOS       | CoreOS | flannel      | [docs](../../docs/getting-started-guides/coreos/bare_metal_offline.md) | Community([@jeffbean](https://github.com/jeffbean))    | K8s v0.15.0
```
Comment (a-robinson) [inline docs/getting-started-guides/README.md]: nit, but can you make this "Uses K8s version 0.15.0" to be more consistent with the nearby rows?

File: docs/getting-started-guides/coreos/bare_metal_offline.md [hunk 2]
```diff
@@ -176,16 +176,16 @@ This is on the PXE server from the previous section:
 
     rm /etc/httpd/conf.d/welcome.conf
     cd /var/www/html/
-    wget -O kube-register https://github.com/jeffbean/kube-register/releases/download/v0.5/kube-register
+    wget -O kube-register  https://github.com/kelseyhightower/kube-register/releases/download/v0.0.2/kube-register-0.0.2-linux-amd64
```
Comment (a-robinson) [inline docs/getting-started-guides/coreos/bare_metal_offline.md]: You don't have to change this, but is there a reason not to use 0.0.3?

File: docs/getting-started-guides/coreos/bare_metal_offline.md [hunk 3]
```diff
@@ -370,14 +377,14 @@ On the PXE server make and fill in the variables ```vi /var/www/html/coreos/pxe-
...
-            ExecStartPre=/usr/bin/wget -N -P /opt/bin http://<pxe-server-ip>/kube-controller-manager
+            ExecStartPre=/usr/bin/wget -N -P /opt/bin http://<PXE_SERVER_IP>/kube-controller-manager
             ExecStartPre=/usr/bin/chmod +x /opt/bin/kube-controller-manager
             ExecStart=/opt/bin/kube-controller-manager \
             --master=127.0.0.1:8080 \
             --logtostderr=true
             Restart=always
             RestartSec=10
-        - name: kube-scheduler.service
+        - name: ≈.service
```
Comment (a-robinson) [inline docs/getting-started-guides/coreos/bare_metal_offline.md]: Was this change an accident?

**Comment by a-robinson** [comment]
Thanks for helping keep the guides up to date! Just a couple small suggestions.

File: docs/getting-started-guides/README.md (hunk 1 above)
Comment (hvolkmer) [inline docs/getting-started-guides/README.md]: I just changed the version, but sure I can change it :)

File: docs/getting-started-guides/coreos/bare_metal_offline.md (hunk 2 above)
Comment (hvolkmer) [inline docs/getting-started-guides/coreos/bare_metal_offline.md]: I was testing the new setup with 0.15.0 and read [this mail conversation](https://groups.google.com/forum/#!msg/google-containers/g9MGseIFXTc/QB4663TdDmEJ). The versiont mentioned there fixed it... I guess 0.0.3 would work as well. I tested 0.0.2 and it worked :)

File: docs/getting-started-guides/coreos/bare_metal_offline.md (hunk 3 above)
Comment (hvolkmer) [inline docs/getting-started-guides/coreos/bare_metal_offline.md]: Indeed. Sorry. Will fix it.

**Comment by a-robinson** [comment]
Thanks! LGTM

**Pull request merged**